debug/src/main/kotlin/io/rover/sdk/debug/DebugPreferences.kt

```diff
@@ -34,10 +35,17 @@ class DebugPreferences(
                 EditTextPreference(
                     preferenceManager.context
                 ).apply {
-                    isSelectable = false
+                    isSelectable = true
                     isPersistent = false
                     title = context.getText(R.string.debug_settings_device_name)
                     summary = deviceIdentification.deviceName
+                    key = "edit_device_name"
+                    dialogMessage = "Set the device name to be used for testing."
+                    onPreferenceChangeListener = Preference.OnPreferenceChangeListener { _, newValue ->
+                        deviceIdentification.deviceName = newValue as String
+                        summary = newValue
+                        true
+                    }
```

Comment:
This dialog is real ugly. bad spacing at the top? perhaps a title/caption message is missing here? Let's make it not too ugly since customers are gonna be using this 😂 
<img width="678" alt="CleanShot 2022-11-17 at 13 59 47@2x" src="https://user-images.githubusercontent.com/16714/202535300-ad4f61d2-7b5e-46b5-9e24-aca06058da46.png">

Reply:
Fixed.